pruning.list

```diff
@@ -70534,6 +70654,8 @@ third_party/nasm/travis/test/vmread.bin.t
 third_party/nasm/travis/test/weirdpaste.bin.t
 third_party/nasm/travis/test/xcrypt.bin.t
 third_party/nasm/travis/test/zerobyte.bin.t
+third_party/node/linux/node-linux-x64.tar.gz
+third_party/node/linux/node-linux-x64/bin/node
```

Comment:
Note: the addition of the `node` binary requires a change in the building process on Arch Linux. Otherwise no issues here.